modules/sfp_threatjammer.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:        sfp_threatjammer
+# Purpose:     Check if an IP or netblock is malicious according to ThreatJammer.com.
+#
+# Author:      diego.parrilla.santamaria@gmail.com
+#
+# Created:     2022-05-03
+# Copyright:   (c) Diego Parrilla 2022
```

Comment:
Please change the copyright to `Steve Micallef`. This is required as I need to retain copyright control in the event that I change from GPL to something else.